cylc/flow/task_job_mgr.py

```diff
@@ -1000,7 +1000,10 @@ def _simulation_submit_task_jobs(self, itasks, workflow):
             itask.waiting_on_job_prep = False
             itask.submit_num += 1
             self._set_retry_timers(itask)
-            itask.platform = {'name': 'SIMULATION'}
+            itask.platform = {
+                'name': 'SIMULATION',
+                'submission retry delays': [1]
```

Comment:
Pretty sure this line was added and removed at least once on the previous branch.

I'm guessing adding it fixes on problem but introduces another?

Reply:
This is a mistake - I clearly got confused yesterday evening, becuase this isn't how I ended up fixing this problem in #5712 .